### DISCUSSION TITLE
vercel-部署-服务端

### DISCUSSION THREAD

**Discussion by owenyk** [walinejs/waline#689] [Q&A] [answered]
请问这些在settings/environment-variables页面没有那三个需要veline配置选项输入框啊，只有name和value，请问是填在那？

**Comment by lizheming** [comment, marked as answer]
文档说的挺清楚了 https://waline.js.org/guide/get-started.html#vercel-%E9%83%A8%E7%BD%B2-%E6%9C%8D%E5%8A%A1%E7%AB%AF
![image](https://user-images.githubusercontent.com/424491/144732521-4c207b5f-ab3e-4475-9e50-3abb101d58c7.png)

**Reply by owenyk** [reply]
> 文档说的挺清楚了 https://waline.js.org/guide/get-started.html#vercel-%E9%83%A8%E7%BD%B2-%E6%9C%8D%E5%8A%A1%E7%AB%AF ![image](https://user-images.githubusercontent.com/424491/144732521-4c207b5f-ab3e-4475-9e50-3abb101d58c7.png)

谢谢，弄清楚了。

**Comment by owenyk** [comment]
谢谢大佬，我弄清楚了，是要新增三个变量和对应值，我以为一个页面能都填上，谢谢
